### PR TITLE
Prettier + Style updates

### PR DESCRIPTION
**Pull request by ericwindmill**

### CODE REVIEW

**Comment by ericwindmill** [comment]
@Levino I added a prettier command to my webstorm. I thought that I already had it set up to Prettier on save, but either way it should be good going forward.

Also, I disagree about one of your comments (making things useful vs making it look nice). It should be both. Users will defintiely change the colors anyway, you're right, but design is important, IMO. Luckily there shouldn't be many design changes needed in future.

Re: CircleCI. I'm not familiar with configuring CircleCI, can you tell me what that failure is because of?

File: .circleci/config.yml [hunk 1]
```diff
@@ -16,4 +16,4 @@ jobs:
             - ./node_modules
       - run:
           name: lint
-          command: npm run lint
```
Comment (levino) [inline .circleci/config.yml]: This change here breaks circleci. The part `command: npm run lint` is in the wrong logic block. The file is a yaml file. Just remove this change and things will work again.

File: src/components/Layout/TableOfContents.jsx [hunk 2]
```diff
@@ -77,7 +77,7 @@ const EntryListItem = styled.li`
 `
 
 const ChapterTitle = styled.h5`
-  font: ${({ level }) => {
+  font-weight: ${({ level }) => {
```
Comment (levino) [inline src/components/Layout/TableOfContents.jsx:80]: Ha, my bad.

**Review by levino — APPROVED** [review]
I fixed the ci issue. Rest is approved! I have nothing against making things nice AND nicely visible. Good job.

**Pull request merged**